src/app/download/download.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="toolbar">
+    <button mat-raised-button color="primary" (click)="downloadFile()">
+        <span>DOWNLOAD AMPHTML</span>
```

Comment:
Since the output is a zip, it'd make more sense to label in Download Zip. Is there a reason why the text has to be all caps?

Reply:
I just made it all caps because the buttons in DV360 are all caps. Should I change it?